cloudslang-compiler/src/main/java/io/cloudslang/lang/compiler/SlangTextualKeys.java

```diff
@@ -46,9 +46,10 @@
     String PARALLEL_LOOP_KEY = "parallel_loop";
 
     //inputs
+    String VALUE_KEY = "value";
```

Comment:
I did want to use the same name, but it didn't seem right.

default is wrong for outputs, because it's not the default value, it's the actual calculated value
value is wrong for inputs, because it's not the value, just the default in case a real one was not provided

Unless you have a good suggestion for both, I guess they will have to be different